w03-intro-backend-with-express/d4-dusk-mongoose/exercises.md

```diff
@@ -0,0 +1,21 @@
+# Express Mongoose Exercises
+
+### Base Challenges
+
+**Adding Mongoose to toEatly**
+
+1. Clone [this repo](https://github.com/sf-wdi-22-23/toEatly_mongoose/tree/master) as starter code. Run `npm install`.
```

Comment:
Change to Clone [our new toEatly-mongoose repo] ... so that students don't get confused and clone modules?